firstvoices/backend/tasks/import_job_tasks.py

```diff
@@ -265,7 +290,7 @@ def batch_import(import_job_instance_id, dry_run=True):
         logger.info(ASYNC_TASK_END_TEMPLATE)
         return
 
-    # If dry-run has not been executed successfully, do not proceed
+    # If dry-run has not been executed successfully, do not proceed for the db import
```

Comment:
Unrelated to these changes, but I just noticed the error log on line 297. Error logs are meant for situations where the code is not running as expected and we need an admin to go fix something asap. This is just a user error being handled appropriately-- I think it should be an info log instead, to avoid noisy alerts for us

Reply:
That log message actually looks like it is intended for the user and not for the logs-- should we put it into the message on the job instead? Or possibly move this check into the API so we could return a 400 status in this case?